Source/WebCore/page/Chrome.h

```diff
@@ -25,6 +25,7 @@
 #include "DisabledAdaptations.h"
 #include "FocusDirection.h"
 #include "HostWindow.h"
+#include "PlatformEvent.h"
```

Comment:
I don't think we need the include here, since we have the forward declaration?

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/c362f4b3787e3a1ab3edc66da8512902bcfb5587